src/content/glossary/index.md

```diff
@@ -679,7 +680,7 @@ The collective [hashrate](#hashrate) produced by an entire mining network. Minin
 
 ### non-fungible token (NFT) {#nft}
 
-Also known as a "deed," this is a token standard introduced by the ERC-721 proposal. NFTs can be tracked and traded, but each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). NFTs can represent ownership of digital or physical assets.
+An instance of one of several contract standards, commonly [ERC-721](developers/docs/standards/tokens/erc-721) or [ERC-1155](developers/docs/standards/tokens/erc-1155). Each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). NFTs can represent ownership of digital or physical assets.
```

Comment:
```suggestion
An instance of one of several contract standards, commonly [ERC-721](/developers/docs/standards/tokens/erc-721/) or [ERC-1155](/developers/docs/standards/tokens/erc-1155/). Each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). NFTs can represent ownership of digital or physical assets.
```

Reply:
"An instance of one of several contract standards"  is super difficult to comprehend, we need better explanation

Reply:
@corwintines i think the order in your explanation is wrong. It should start with explanation of uniqueness and end with saying there are few versions of these standards.

Reply:
NFTs can represent ownership of digital or physical assets. Each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). There are two smart contract standards used for NFTs, [ERC-721](/developers/docs/standards/tokens/erc-721/) or [ERC-1155](/developers/docs/standards/tokens/erc-1155/).